alerting/src/test/kotlin/org/opensearch/alerting/TriggerServiceTests.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting
+
+import org.apache.http.entity.ContentType
+import org.junit.Before
+import org.mockito.Mockito
+import org.opensearch.alerting.model.BucketLevelTriggerRunResult
+import org.opensearch.alerting.model.InputRunResults
+import org.opensearch.alerting.model.MonitorRunResult
+import org.opensearch.alerting.script.BucketLevelTriggerExecutionContext
+import org.opensearch.common.xcontent.DeprecationHandler
+import org.opensearch.common.xcontent.NamedXContentRegistry
+import org.opensearch.common.xcontent.XContentType
+import org.opensearch.script.ScriptService
+import org.opensearch.test.OpenSearchTestCase
+import java.time.Instant
+
+class TriggerServiceTests : OpenSearchTestCase() {
+    private lateinit var scriptService: ScriptService
+    private lateinit var triggerService: TriggerService
+
+    @Before
+    fun setup() {
+        scriptService = Mockito.mock(ScriptService::class.java)
+        triggerService = TriggerService(scriptService)
+    }
+
+    fun `test Run Bucket Level Triggers`() {
```

Comment:
Can we name this test to `test run bucket level trigger with bucket key as int` to be more specific.

Also, did you happen to check something like a composite aggregation containing an aggregation on int? In that case we'd hit the `bucket[keyField] is Map<*, *>` case and convert it into a String which I'm assuming may behave strangely and not be fully supported?